src/navigation/nav2_bt_navigator/CMakeLists.txt

```diff
@@ -18,19 +18,26 @@ find_package(nav2_planning_msgs REQUIRED)
 
 include_directories(
   include
+  /usr/local/include/BTpp
```

Comment:
Same comment as other CMakeLists files, need to use CMake find_library for the BTpp library